models/OrderModel.ts

```diff
@@ -41,4 +41,8 @@ export class OrderModel extends BaseEntity {
       items: this.items.map((oi) => oi.getPublicOrderItem()),
     };
   }
+
+  public getItemsAndQuantity(): MerchItemOptionAndQuantity[] {
+    return [];
+  }
```

Comment:
What's this method for?

Reply:
OOOPS, my bad, this was for a prior implementation that I realized we didn't need, I'll delete it